src/modules/flow/http-client/http-client.c

```diff
@@ -584,6 +591,8 @@ int_process_data(struct sol_flow_node *node,
 {
     int value;
 
+    SOL_HTTP_RESPONSE_CHECK_API(response, -EINVAL);
```

Comment:
No need to check the version here, it's known to be good since that is checked before the packet is created.